benchmarks/ios/visual-effects-ny/shared/src/commonMain/kotlin/HappyNY.kt

```diff
@@ -17,8 +17,8 @@ import org.jetbrains.compose.demo.visuals.platform.nanoTime
 import kotlin.math.*
 import kotlin.random.Random
 
-const val width = 400
-const val height = 800
+const val width = 640
+const val height = 480
```

Comment:
Will it display properly on iPhone?
Maybe we need some expect / actual getters

Reply:
Good catch. Added expect/actuals